<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7105" started-at="२०१७-१०-१५T०७:५०:०८Z" finished-at="२०१७-१०-१५T०७:५०:१५Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7105" started-at="२०१७-१०-१५T०७:५०:०८Z" finished-at="२०१७-१०-१५T०७:५०:१५Z">
      <class name="Reporting.extentreports_screenshots">
        <test-method status="PASS" signature="verifyTitle()[pri:0, instance:Reporting.extentreports_screenshots@5e3a8624]" name="verifyTitle" duration-ms="5475" started-at="२०१७-१०-१५T१३:२०:०८Z" finished-at="२०१७-१०-१५T१३:२०:१४Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitle -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:Reporting.extentreports_screenshots@5e3a8624]" name="tearDown" is-config="true" duration-ms="1204" started-at="२०१७-१०-१५T१३:२०:१४Z" finished-at="२०१७-१०-१५T१३:२०:१५Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyTitle status=SUCCESS method=extentreports_screenshots.verifyTitle()[pri:0, instance:Reporting.extentreports_screenshots@5e3a8624] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="invalidtitle()[pri:1, instance:Reporting.extentreports_screenshots@5e3a8624]" name="invalidtitle" duration-ms="29" started-at="२०१७-१०-१५T१३:२०:१५Z" finished-at="२०१७-१०-१५T१३:२०:१५Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at Reporting.extentreports_screenshots.invalidtitle(extentreports_screenshots.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidtitle -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:Reporting.extentreports_screenshots@5e3a8624]" name="tearDown" is-config="true" duration-ms="368" started-at="२०१७-१०-१५T१३:२०:१५Z" finished-at="२०१७-१०-१५T१३:२०:१५Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=invalidtitle status=FAILURE method=extentreports_screenshots.invalidtitle()[pri:1, instance:Reporting.extentreports_screenshots@5e3a8624] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Reporting.extentreports_screenshots -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
